engine/runtime/src/test/java/org/enso/compiler/EnsoCompilerTest.java

```diff
@@ -1067,7 +1066,24 @@ public void testGroupArgument() throws Exception {
   }
 
   @Test
-  @Ignore
+  public void testRuntimeServerTestCode() throws Exception {
+    parseTest("""
+    from Standard.Base.Data.Numbers import Number
```

Comment:
We need to find out why `RuntimeServerTest` is failing. I thought it was because of wrong UUIDs in the `import`, but fixing them doesn't help. CCing @4e6.

Reply:
Morning investigation suggests that one elements doesn't get an `externalId` assigned from metadata. Investigating.